components/ai_chat/core/browser/conversation_driver.h

```diff
@@ -10,6 +10,7 @@
 #include <optional>
 #include <string>
 #include <string_view>
+#include <utility>
```

Comment:
this addition doesn't seem related?

Reply:
this was left out in one of your PRs (conversation api one?) which resulted in the following format issue

> ../components/ai_chat/core/browser/conversation_driver.h:147: (cpplint) Add #include <utility> for move [build/include_what_you_use] [4]